src/main/java/com/quadfour/dao/ITaskDAO.java

```diff
@@ -4,10 +4,10 @@
 import com.quadfour.dto.TaskDTO;
 
 public interface ITaskDAO {
-
-    boolean save(TaskDTO taskDTO) throws Exception;
+    // I know its nitpicky but this should probably be renamed to saveTask just to be clear what you are saving
+    boolean saveTask(TaskDTO taskDTO) throws Exception;
```

Comment:
I'd keep this as it were.  saving a task is a given, based on the explicit parameter type.  That's adequately self-documenting.